utils/gas-utils/gas_utils.go

```diff
@@ -32,8 +31,8 @@ func CalculateReceiveMessageGasLimit(numSigners int, executionRequiredGasLimit *
 	gasAmounts := []uint64{
 		executionRequiredGasLimit.Uint64(),
 		ReceiveCrossChainMessageStaticGasCost,
-		uint64(numSigners) * ReceiveCrossChainMessageGasCostPerAggregatedKey,
-		ReceiveMessageGasLimitBufferAmount,
+		uint64(numSigners) * warp.GasCostPerWarpSigner,
+		warp.GasCostPerSignatureVerification,
```

Comment:
this is cost to verify the aggregate signature, and doesn't matter how many individaul sigantures were aggregated right?

Reply:
warp.GasCostPerSignatureVerification is the fixed cost the Warp precompile charges for each verification request, and warp.GasCostPerWarpSigner is the amount charged per signature in the aggregate signature.